limits.c

```diff
@@ -409,6 +405,6 @@ void limits_force_servo(){
   
   plan_sync_position(); // Sync planner position to current machine position for pull-off move.
 
-  limits.mag_gap_check = MAG_GAP_CHECK_ENABLE; // Start checking magazine gaps on carousel again
+  limits.mag_gap_check = settings.mag_gap_enabled; // Start checking magazine gaps on carousel again
```

Comment:
No, it's not just an alias. limits.mag_gap_check gets set to 0 on some occasions and set back to settings.mag_gap_enabled on other occasions. limits.mag_gap_check therefore does not always equal settings.mag_gap_enabled.